pkg/apply/apply_test.go

```diff
@@ -93,8 +93,8 @@ func TestApplyBasicUpdates(t *testing.T) {
 	topicInfo, err = applier.adminClient.GetTopic(ctx, topicName, true)
 	require.NoError(t, err)
 
-	// test NewOrUpdatedChanges has expected shape when updating configs
-	expectedChanges = &NewOrUpdatedChanges{
+	// test Changes has expected shape when updating configs
+	expectedChanges = &Changes{
```

Comment:
As you are fixing a bug that produced output when you were not supposed to produce output, please add a unit test that verifies that condition.
Adding a non regression test that tests a specific condition you are trying to fix is a good practice as, since this broke once before, it is likely to break it again. Adding the specific test makes it impossible.

Reply:
added a test for no-op apply changes